apps/designer/server/prisma.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Instantiates a single instance PrismaClient and save it on the global object.
+ * @link https://www.prisma.io/docs/support/help-articles/nextjs-prisma-client-dev-practices
+ */
+import { PrismaClient } from "@prisma/client";
```

Comment:
to import {PrismaClient} from "@prisam/client";
,need to run command: pnpm exec prisma generate in the dir vvplus/apps/designer/